### DISCUSSION TITLE
No space left on device

### DISCUSSION THREAD

**Discussion by corrcorr** [Security-Onion-Solutions/securityonion#4457] [Unsupported Versions]
Hello,

i need help with managing disk space in SO standalone installation, after running for sometime the disk get to 100%.

**Comment by facyber** [comment]
Hi @corrcorr 

You will need to provide more information than just that you run out of disk space.

What version of SO are you running?

Check what data is taking disk space in /nsm folder? Use command `du -hs /nsm/* | sort -n` and paste output here.

**Reply by corrcorr** [reply]
Hello @facyber 
I am using SO 2.3.30, below is the output 

[bloom@bloomso ~]$ sudo du -hs /nsm/* | sort -n
[sudo] password for bloom:
0       /nsm/import
0       /nsm/pcapout
0       /nsm/soc
0       /nsm/suricata
2.3M    /nsm/thehive
4.0K    /nsm/logstash
4.0K    /nsm/pcaptmp
7.7G    /nsm/pcap
13G     /nsm/docker-registry
28K     /nsm/osquery
96M     /nsm/strelka
162G    /nsm/elasticsearch
236M    /nsm/mysql
267M    /nsm/wazuh
277M    /nsm/pcapindex
541M    /nsm/zeek
613M    /nsm/backup
683M    /nsm/repo
720K    /nsm/grafana
751M    /nsm/influxdb
[bloom@bloomso ~]$

**Reply by corrcorr** [reply]
also please note that I have more than 500 GB of HD, 15GB RAM and 4 processors on the VM

**Reply by facyber** [reply]
Check what is your `log_size_limit `value (not sure where it is on standalone, but on distributed version it is in `/opt/so/saltstack/local/pillar/minions/$minion.sls`)

Check Curator logs to see if it works properly and cleans the Elasticsearch indices. If Elasticsearch data reaches `log_size_limit`, then Curator should delete those old indices.

You can also check Elasticsearch logs just to see if t hey are giving you more information.

**Reply by corrcorr** [reply]
i cannot seem to access the /so director, this is what i get when navigating to the directory  even if i run it with sudo

[bloom@bloomso opt]$ cd /so
-bash: cd: /so: Not a directory


Curator status keeps showing pending, when i stop and start it then it throws the error below

----------
          ID: so-curator
    Function: docker_container.running
      Result: False
     Comment: Created container 'so-curator'. Failed to start container 'so-curator': 'Error 500: failed to create OCI runtime console socket: mkdir /tmp/pty279652655: no space left on device: unknown'.
     Started: 13:28:02.550271
    Duration: 1757.661 ms
     Changes:

**Reply by facyber** [reply]
You need to check in your folder of installation, see where it was installed (this was an example from distributed setup, on Standalone not sure if it the same structure).

Check older logs to see if curator ever worked, also suggest you to delete some indices if you don't have important data.

If it is an option, please try updating it or installing latest SO version.

**Reply by corrcorr** [reply]
This is what have in indices, so do i have to delete the directories or what. How do i go about deleting elasticsearch indices?
thank you in advance 


[bloom@bloomso indices]$ ls
03_Ilyy3SoGe04FtSyqc5Q  4_qHUIYxQq-IUWC_rjIDOg  9Z6tQ79USVS4K5Auq-8ioQ  _EBnH9lxT2OpDCUZBKpUXg  iaRzdFgOSjCWhkmoIXKm4Q  _lNdep8-QC2cLAfX68nCEg  oHpcX17vTI2PVphobYoM3A  TCCsNLHmTmanxexPguVIDA  w36YuIdbRYePAZD67sGlSQ
044WkJQzSYuZjysuHbbnag  4WrMQniKQsC4ddKFo09p5w  a1qSnEf-QGi8UILerbPUeg  EcSLqSwqQGm14I9ZAva8Vg  iBfx3FnoSzydqYl13PnnNw  l_oo3q_4SVO-tjGA4XDUQg  OmV4pya1RgqextBjIlK8bQ  tEFxmda0TV-gJgvEWwwx6Q  wE8GlUtuTRmnVQhufAGtPw
053gmOAkQxit73Yc1K2iDA  53xEWmEKRoOI25wL6IhD4g  a6QyfRkqQGKeJMPTZBT9tQ  EeNN1dFsSBuvkyQ8DA2u2g  iFUkRZZKR7eXxbYc4oRX1A  luinr-xzTsqEwWoRwO23Fg  OPYGSXX2QCicXwNpvYiFvQ  Tl7NPMTWQrakepathMAy_Q  WON-qFt6QMeAHSt78nOS0w
0DBfOADQTcO8kESwMU6P6w  5c01GgD5Qm-_F5N5X87zXQ  a9hFsbNhQxiWy-AEldY-Ug  EPajdMp2QwqMG3kgGmBFaA  IJINI1cNRRO6PQe90Gi92A  LVRPE_DGSCCzRKM8m6swMw  OZIWPg-gRH-3xN5kW5eY4w  t_N0BuWjSlS5fZifzLArLA  X3rJSnX2TbehEo-5-AV_9g
0HVP2KIrQPmnul-aJ8qnpA  5CVHyKc3T9eDB4nsFOTtjQ  AALOT3reSGu9VB2dzg2lJA  Er_9eSd-RIKX5GKnWcqBUQ  iMaExn7TRjGi3KmHWeXxgg  Lwrj7Q1BRT-ks52Rpyg8ug  p01g2SR4R-mhP0XOn5zBRg  Tp-m_EFDShGRyZRR6KtPcQ  x5dYG-wQQ5m9-3PoIPqlig
0lp799C7S_eLbICLML1oYg  641xPOJrQWyalemIvcaDVw  abQso70zRuaAo6_i4bLtag  eRnRL5MOR02nZQIwP3Sr8A  i-OK2OLkRNaHQIccD0sofg  mA7PqF8BTwGZXI2rjmIGWA  PdRLWYwLRLOcK7g7AQ4awQ  TvuNVySzSz6280rccgAq1Q  xF3Tm_5XTRijVm6vfUroSA
0o6i-EMSSY-9im3fbFXOFw  6IlKty57RQqdL0wvucuU5w  ACjKcXz2Rwm_1XZbTqqsoA  ETopdThfQX2HOK4Szb0AzA  ipHK8hpST-eS5-h01pScdA  Mdkb08vxQGWxhxPgGIG0Dw  pFlbY0CdRMOY4hQgxU3fCw  TXIEL_RJQMmigYAhin2Ysw  xjGTELo0TXiL5eISlD_i-A
0rJuOJcmRSa-DLgH_nFyoQ  6JOXWxSJRkilDWxiYEJMMA  Ad3apOCIRcue9zCEVOR_mQ  fAEZ6IHxSkCBZzI077rfHA  Iwma3cP8S-SNSfkMLZ9FXA  Mf_YY2x8TKy32QlGGjxr0g  pH11uhQZSfGYFnwe8g7S_w  U0gL8nDrQgu_PIDmPEhAgQ  XJtIKGXyTkeuDYD7iAh2yg
0W-k2BDIRA6guUYdSNbinA  6KwkEarFQv6_jU9FJJT44Q  ADhD476ESVi8zNvHkuuIeg  fdgKRZa1SkGilQEHbib-2g  IzLFUcDFRvOonosOk__caQ  mJ4AqzD4SQ--RTWXVEMfWA  phAnzlvbTHmUKdV1A98Tsg  uaGh0DwZQniaXEuuNZBF_Q  xnj3J1TRTz6N0OXgx6E20g
13tcQMZYSpqJDDdgfjNUCg  6r5cgMhkSkuITA0wZuY2Tg  AeNTq6TPRziSyojGA1L6Lg  FgF-z0txSHiSDmVPVWVB-Q  J2CeBSSFSZSjuud_VI62SQ  Mlg_aB4kRn6lXLHYUm7VhA  phdut3DeQPSJf86QhCdH2Q  udLFdcj0SCah1JOBzIZLyw  Y1TFWG_NSCCszQRzNU7OoA
18BSVmrwRPO9WvCi1fWxfg  6sOQy5VUTGGCvrbgnDLz1Q  afhUYTkhT7uM5e4-pUHIrA  fPYLyc7qQRCGdTkLmkKb-Q  j71K9AVhTHiG9aZuZY1kag  -mNQNJ3rRI67byUgTufYuw  PJoRUtNMQkSe4KOcA9jgEA  uGeljYwlSiWhR2jUNLTNYQ  YE66n6oYQyuZoFOymsfJyg
1e1Y1X6eS-is2O0TH3mHwA  6YPXpb85SWaNOXAcJK48iQ  AFStJjqnSHCG0cfKpMHPJg  fswIjvmhQXGcMewHOtjzbw  j-9sOqC8TnyMMlgzMSamfg  Mpdtffk9TDSUC9yRQnn4Rg  pK6LCfXuTSW-ez4FFxNZEQ  ujMwyaBCT-iVSE99XcGgoA  YI_55uk9QyK2VQmMKJZWfw
1IUiPIJ8Q-KMP6smpXonVw  75uYWgWzSdqoEIyrUvQtkQ  aifkFT0bRTGeO08uPT8KUQ  fwppiZRgRDeG20VjOXKQsA  Jg65K6zqSHClerxAzR_wmQ  MqnCZfy_R2OvuFjZ-CLzlQ  PKIqv_DpSvujbkpJjVwyKQ  UO9UqL5EQnG9ID3hBSnh5Q  yo9igAoIR-W5QkUukoyagA
-1KNH0-gTe-DgRQ5XmmWDg  7AxkE4Z5SnGQjD0QfML70A  aLkmhg30T4Kq4oTP-pig0w  GACkcCCgR0agF5CG2BHIcg  jGc6kIPxSGSY7KEJfhrrWg  mtuhIWAuQVyTN4kA075QAQ  qERWLnWpT46OIgvQgM3OCQ  up_sDtDaSPWeD_VDsOT3_Q  _yUG-a2rTgiqqKbU5LQ55Q
1kSbRBPUSg-JaDDpZcNoDQ  7b9VGkSeStGVX102b_SmjQ  anjKuVaqTJOE6b4SSkci6w  gLKX5maXSuessNCkZMV4zw  JicHXLtPRTal1o_IyctVNw  _n0AwSpyRty9LzJtIrWbjw  qgCxjaYfQc6GMjvZi2DsWw  UR3mP8WuRVq97uyqJtb6rA  Z8qPiRfrT6yKfZtIW36E7g
1WAr1CQPSu60JbHI2J2Zog  7g5nDYvQT6KP8CtrRNXpBw  AxcY4CotSQuqEBi8Ibzm6A  gM2CGcAMRg-cqiHWFpwQTQ  Jpqt1RLPSEicLqNPGcz4Pw  N4K5hJ8ESMOYnBd5WQN1iQ  QmCwxEg0R56qg0WgNfTpfA  UUg1kh40RTi9s4_90WmZ1w  ZaVpRkODQEutgKixCOb9Dg
1Wi9_iMAT4ydbof75xYzUA  7GH355ybTQGKxTzBMkaVng  bVmzm4PGQ6KS_JbaFmfjUw  GUvAUcpES7S9bT2YAww9vA  -J_SvY3pS8eFVfiobmYmRg  N6Z2SruAS8yA39RVLrCYuw  qUf5bYwuR1KrnSqjZN3rgg  UUWp8MnVR5KvgXzjdpCXXw  ZAWDtNiGSIyG4x4qpClglg
1X9gLUrVShaNJMNzizYYzQ  7oRjmGl5Qqm1oI09tAU5OQ  CauIvJjyROO04I3rtDmoJA  GV-ZSPHzQeSAlrjRHICsyA  JtHmJ0ovT-m1QLXZrDqiIg  _ndRVw5JSbC7YNygSMmztg  qUuBuZmbQ9-Sn4x6o6YiSg  uYKKDEKBTt2Meqf3FKWf3w  ZdSDr7m0Qlq261kfN0WEew
21LozWuISueTv6H8i9A6Wg  7QixHKp6QvuuPymKa6Owow  CbOzJ82VQG2jlGgM6vEBDg  gYl8E4z7RAK5D3UZwbUW2A  jVvGv0eFR9OMv_ECLJHX7Q  nH9vlksBRDeBI6IuL6zQwg  QWldbdJRRpGv6CWSLHLYAQ  V0ZD7BYiSrSOh6j9IILReQ  ZdwKr4wMRaGImVNh4Rwd0A
25pVRhyiTQiXsyAHIZk6dQ  _7rMkJvPTw2UCqTUyKHQcw  cc1Blt9STsuqhqMwC7kchg  H3k0Jx3RSve5s0OGWvGygg  KBovNIQoSWi21S_Qw4sBoA  NnLkHt_bQYmtGoG9qyqvPQ  qYpsBJSCS8eLfVU6QzobVg  V3rSDQ8HS-iSyRTjkmAXQQ  ziAfJxpUSn69IuI3ydrz7w
2hafhPQuSOqkuYQbRlz11g  7RyqXfmxR7WUA36zh_yM3g  CHpbqNOoR9uCyfNxyNTMhQ  H-4StdP_TfaIvo5Co4HT-A  KCeb15PuQCm01zWztQEv8w  nQ81Zb1MSsaxgYiSPn3mjQ  R59ZDWooTwWDOy79KO5r6g  V8XpwjfNQqGRTdzO-iPJ3Q  zjgWEifPSZ2Wm8X0vDZfpg
32nK3uwAQtKxbOGXhaC_eA  7utxLVRaRLOLCM-zXz-qbA  C-hVkYaDQnGeGLKkbp45wA  H9BeyjbQSXSM2ujuTv7iAQ  kCjrwzf5TfGzRy1ugEdlYw  NQhi6WTnRBW0C0MOK4ne7Q  RcX5wLMSR8urNjca57OmYQ  vF32AcsDTjS26D3yBbovZg  zKcJxOMLSZ-2RtylTerlxg
32YowGObTBK5Lw58E5Dzaw  83cI2VaoRs2rNAP0oqurxQ  d9ouEPVwTmaaobS9EC7oVg  HCFwvjX4QN2LsDURXYFh4g  Kg7BnIqZTA2jTyBI3To1DA  ntOKNHwGQQ23cur-6GOw8A  RdIRnXwITBKJo4HWtLLMYQ  VI8uWZ3UQaOkh3ny2rg7bg  ZPRiGLv9RpCdGRbzT2KKlQ
34vjTHr4Rs-B0LbnaqZouQ  8Bi3ST7oTEm2lO4xWYUWSA  DarClIdERJO_8yTJ8wHleQ  Hfddge9FQiaVU-N7jSKRhg  KOMIK5zlRsOjR2B6L9Fd4g  nu0LDy0jRFW-5sQfhl6kdw  r_gUlMptTe6tT1oWY2OoUA  V-j4LIpVSSeI037Y-4EXFg  zqqVJOIcRjietlIdi6VO0Q
3gYH72bVTmqteM8ptg8e4A  8feJ4MYES5KKUzOajwIX5g  df6kl4ZfRI6DsQytI6OQvw  HJvFMH4oSYihQrMXcuKzew  KpRPDA3LTICF70bpt__8Iw  NUBNVifHRw6csRCM9HvABA  RLYA6Z_8SPayNBWdI7dfHA  vJK6oat9S7ir99ftZgH8GA  ZRmLwcGdRBaeZKe1lKcqCg
3H3wtSvLQEGzLM7eUJymRg  8hbdoO9JQLaRQBL3y1t5EQ  DLIFD2A0Sx-v4XKmF4sIYQ  HMCzb55HSNKRvz4eM9ws7A  KpZ2aB9TQd6VivnmB8yLlA  o1_N2hvVT62a0rX1o1IqiA  RNU4-SYpReu4tfR3BoTi-A  VLYFttz8TGm4h9zQHZL51A  ZxRgIiKQSEqeF_zLjMHO5w
3J98w2tOQpqWVAxtjm2DQg  8JdJEAHrQ_Gyx9mjo5AN4g  dovrjZfvTRWOSsjn9unfAQ  hODTJU0XRXachTflABOv4A  KtM7Z534Tgm-n0HMBioY-w  -O4AQTEYRaurSf1Tqo5F1w  sITtHmVLTuqlqJGX8BfosA  vMTaOejdRcKG6NNiu7quiA
3NhrCRwyQi2bQfCOhfJHTg  9cKoGLSqQwuv4drjEKHGrA  dSpnOej8ToiXnrIMNax6sg  hqt2mUv5QXuV1ccOgRcPYw  Kyh3vf5WRtartoSc_CNCbg  o_4nkPzsQuW4kaEv3qUklw  sWhNXbtPRuCYq-93GSdjog  _vNz9hNWQ26msjlkj6Bm3A
3W7ZD3q_Q_a0TJSiIZ7CQg  9pZvT0wrRjOZjSzdAGUXfg  dVyokHNNQTKlaOeNIpshbg  hVfBAnNITm2N44mfj2NppQ  L2DFaR8ZRmqporh8WfhycQ  o4TkzDZ8Ttm7mGw1nhi4PA  szE6_PuVRXCO35flTDrdVg  VRlA2ke3QpOD9wzKdq0EjA
469id6gEQiynReCeReWsrw  9r5jkeSERJaOwG0gWz9oZA  dWabs1H7QQu1JPWwyMrjSg  HVtP_NrLSr6LxOkg_5-8xA  L30SHZ-NSIWM_IfE-yNQ-A  o69O-5MMSKC2qhUikgmPHA  t0fClkl7RkiQaKGPRuTRyw  w01Qw4wAT_-k8QwIuTke4A
4c-jfcY2QZ-QPAPfWNcCzA  9Vcvr-spT52nc1ZPR5MYGA  -e8oS2csQqOnowkNSXC01w  hyBy0tP4QMms85j6NXIUbw  lCDqg3kcTdyGXjBBgoDM3Q  OaVTI0SYRS-0kUiOX5w9hQ  T8tsWcMBSIOqi50swcUO-A  W0o6uppkRJutTnrVUfXZ-w
4nJ67d3VTBiTNPnnJkl2pQ  9YMQ0xygTyCbRYQJQ2crDQ  EaipdBQBReSVVtUG8FtnVQ  I6b7LvNNTEuMKyVHtkwjhA  lGoDBiX4TWGIc-7kY9WHhA  -_O-DxuKRf-5z_wAKNeGTQ  tA6Ko3SJSMqa8fA4a5VgtQ  w0tVCDALRWeX2JGcKs_yug

**Reply by facyber** [reply]
I suggest you go through the [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html) and some youtube videos on Elasticsearch basic commands and management as you will need them for SO. Its mostly basic Linux commands (which will you also need).
